### DISCUSSION TITLE
resolve 1.20.1 - dependency - cypress open application error

### DISCUSSION THREAD

**Discussion by michalbudzynski** [cypress-io/cypress#19539] [Plugins]
The "resolve" library was updated 15 h ago.
https://www.npmjs.com/package/resolve
There are problems running cypress application with node_modules dependency `resolve` ver. installed: 1.20.1
After opening the cypress application `cypress open` - the application shows an error.
```
The function exported by the plugins file threw an error.
We invoked the function exported by /cypress/plugins/index.js, but it threw an error.

TypeError: Cannot read properties of undefined (reading 'uid')
```
If anyone has the same problem: the solution is to install the previous version 1.20.0

My index.js files contains:
```
const browserify = require('@cypress/browserify-preprocessor');
const cucumber = require('cypress-cucumber-preprocessor').default;
const resolve = require('resolve');

module.exports = (on, config) => {
    const options = {
        ...browserify.defaultOptions,
        typescript: resolve.sync('typescript', { baseDir: config.projectRoot }),
    };

    on('file:preprocessor', cucumber(options));
    require('cypress-mochawesome-reporter/plugin')(on);

};
```

**Comment by IndumathyBabu** [comment]
Hi [michalbudzynski](https://github.com/michalbudzynski)

 I am facing the same issue only when running using Saucelabs and not in local. Please help me with this
